### ISSUE TITLE
ERROR: Exception caught during firing event: STONE_TORCH

### ISSUE THREAD

**Issue by Wissi** [bl4ckscor3/CeilingTorch#19]
Awesome mod, thank you very, we love your mod and hard work, we include it in our modpacks always.
The latest version update crashes the client.
```
[modloading-worker-1/ERROR]: Exception caught during firing event: STONE_TORCH
	Index: 1
	Listeners:
		0: NORMAL
		1: ASM: class bl4ckscor3.mod.ceilingtorch.CeilingTorch onInterModEnqueue(Lnet/minecraftforge/fml/event/lifecycle/InterModEnqueueEvent;)V
java.lang.NoSuchFieldError: STONE_TORCH```

Crash log [here](https://gist.github.com/Wissi/dd6f5d199bc365052e2b35b874fc9d48).

Using:
Forge 3.2.30
ceilingtorch-1.15.2-v1.5.2.jar

**Comment by bl4ckscor3** [comment]
What version of Silent Gear are you using? This is the exact error that I fixed in v1.5.2 of Ceiling Torch (see #18) and it should not happen if you are using the latest version of Silent Gear (1.8.2+109 at time of writing).

**Comment by bl4ckscor3** [comment]
Closing this as resolved. If you have more information on this and are using the last version of both mods, feel free to comment.
